protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1727,6 +1727,10 @@ message ListTransfersRequest {
   optional Pagination pagination = 3;
   // Restrict transfers to those related to reward transfers.
   optional bool is_reward = 4;
+  // Epoch to filter from (included). If omitted, `to epoch` must not be set. If both omitted, the most recent epoch's data is returned.
```

Comment:
Any reason why this doesn't work like any of the "range" filters in other API, and allow a `to` to be set without a `from` and you just get everything from `0` up to the `to`?

Reply:
We made this choice because we couldn't find any reason to restrict this.